src/calculateRentalCost.js

```diff
@@ -28,7 +28,23 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let cost = 40;
+
+  if (days < 3) {
+    cost *= days;
```

Comment:
В твоём случае можно сделать так:
```suggestion
    return count;
```